templates/workspace_services/databricks/.dockerignore

```diff
@@ -0,0 +1,4 @@
+# See https://docs.docker.com/engine/reference/builder/#dockerignore-file
```

Comment:
Files are missing here. Check the base workspace for an example.

Reply:
I've replaced the content with the file from base workspace